models/user.go

```diff
@@ -65,7 +65,7 @@ func NewUser(instanceID uuid.UUID, email, password, aud string, userData map[str
 		InstanceID:        instanceID,
 		ID:                id,
 		Aud:               aud,
-		Email:             email,
+		Email:             strings.ToLower(email),
```

Comment:
this is a good idea to ensure that future accounts created are always case-insensitive